docs/guides/integration-prebidjs.md

```diff
@@ -0,0 +1,193 @@
+        subscriptionId: '...subcription id...',
+        email: 'user@example.com',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will normalize and hash the email address before sending the encrypted hash to the UID2 service.
+
+### Configure for phone
+
+Configure the UID2 module with a phone number:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        phone: '+1111111111',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will hash the phone number before sending the encrypted hash to the UID2 service.
+
+The publisher is responsible for normalizing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md#phone-number-normalization) for details on phone number normalization.
+
+### Configure for hashed email
+
+Configure the UID2 module with a hashed email address:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        emailHash: 'eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will encrypt the hash before sending it to the UID2 service.
+
+The publisher is responsible for normalizing and hashing the email address. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for details on email address normalization and hashing.
+
+### Configure for hashed phone
+
+Configure the UID2 module with a hashed phone number:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        phoneHash: 'eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will encrypt the hash before sending it to the UID2 service.
+
+The publisher is responsible for normalizing and hashing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for details on phone number normalization and hashing.
+
+## Check the integration
+
+Check that Prebid.js and the UID2 module are configured correctly by calling `pbjs.getUserIds().uid2`. If everything is working, this will return the UID2 token.
+
+## Reconfigure the UID2 module as required
+
+If a user is visiting your site frequently, the UID2 module will be able to continually refresh the UID2 token.  However there are cases where this is not possible.
+
+A UID2 token can only be refreshed for a certain amount of time. If a user returns to your site after this amount of time, you will need to re-configure the UID2 module.
+
+To check if the UID2 module needs to be reconfigured:
+
+```js
+TODO
+```
+
+## Optional: Set the UID2 API Base URL
```

Comment:
Rephrase to something like:

_Optional: Reduce latency via Geolocation-based API Routing_

Saying _Set the UID2 API Base URL_ is a bit verbose and easier/best to say what the result is for

Reply:
Doesn't "Geolocation-based API routing" refer specifically to the AWS Global Accelerator, and not the region-specific URLs?

How about "Optional: Reduce latency by setting the API base URL"?

Reply:
I don't think it refers specifically to AWS GA, but how about

_Optional: Reduce latency by using the API endpoint closer to your readers_

Reply:
I like that except "[endpoint](https://unifiedid.com/docs/endpoints/summary-endpoints)" has a specific meaning in the docs.